index.bs

```diff
@@ -7,7 +7,6 @@ Group: WICG
 ED: https://wicg.github.io/local-font-access/
 Repository: WICG/local-font-access
 Abstract: This specification documents web browser support for allowing users to grant web sites access to the full set of available system fonts for enumeration, and access to the raw table data of fonts, allowing for more detailed custom text rendering.
-Editor: Emil A. Eklund, Google Inc. https://google.com, eae@google.com
```

Comment:
Bikeshed supports "Former Editor" - I don't know how this renders, though. Try it?

https://tabatkins.github.io/bikeshed/#metadata-former-editor